blockcode/README.md

```diff
@@ -418,25 +437,36 @@ Now we can use the functions above, with the `Menu.item` function from `menu.js`
 
 ## Lessons Learned
 
-### Why not use MVC?
+### Why Not Use MVC?
+
+Mode-View-Controller (MVC) was a good design choice for Smalltalk programs in the '80s and it can work in some variation or other for web apps, but it isn't the right tool for every problem. All the state (the "model" in MVC) is captured by the block elements in a block language anyway, so replicating it into Javascript has little benefit unless there is some other need for the model (if we were editing shared, distributed code, for instance). 
+
+An early version of Waterbear went to great lengths to keep the model in JavaScript and sync it with the DOM, until I noticed that more than half the code and 90% of the bugs were due to keeping the model in sync with the DOM. Eliminating the duplication allowed the code to be simpler and more robust, and with all the state on the DOM elements, many bugs could be found simply by looking at the DOM in the developer tools. So in this case there is little benefit to building further separation of MVC than we already have in HTML/CSS/JavaScript.
+
+### Toy Changes Can Lead to Real Changes
```

Comment:
I just noticed it says "Mode-View-Controller" and should be "Model-View-Controller"

Header changes are fine.